### PR TITLE
Pager.js: allow to set more classes

### PR DESCRIPTION
**Pull request by sergiocarracedo**
Add more properties to Pajer.js component to allow users to set customs classes in navigation elements (First, Prev, Next, Last)

New properties
* firstClass: Custom class to the link to first element 
* prevClass: Custom class to the link to prev element 
* nextClass: Custom class to the link to first element 
* lastClass: Custom class to the link to last element 
* navClass: Custom class to all navigation elements (this class doesn't replace other custom classes

Add .idea/ to .gitignore

### CODE REVIEW

File: gridsome/app/components/Pager.js [hunk 1]
```diff
@@ -49,16 +54,20 @@ export default {
         linkProps.exactActiveClass = props.exactActiveLinkClass
       }
 
+      if (!Array.isArray(linkClass)) {
+        linkClass = [linkClass]
+      }
+
```
Comment (hjvedvik) [inline gridsome/app/components/Pager.js:60]:
```suggestion
```

File: gridsome/app/components/Pager.js [hunk 2]
```diff
@@ -49,16 +54,20 @@ export default {
...
       return h(Link, {
-        class: props.linkClass,
+        class: [props.linkClass, ...linkClass],
```
Comment (hjvedvik) [inline gridsome/app/components/Pager.js:62]:
```suggestion
        class: [props.linkClass, linkClass],
```
Vue merges arrays and objects into class names internally.

File: gridsome/app/components/Pager.js [hunk 3]
```diff
@@ -49,16 +54,20 @@ export default {
...
         attrs: {
           ...linkProps,
           'aria-label': ariaLabel.replace('%n', page),
           'aria-current': current === page
         }
       }, [text])
     }
-
+
```
Comment (hjvedvik) [inline gridsome/app/components/Pager.js:70]:
```suggestion

```

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,4 @@
 node_modules
 coverage
 dist
+.idea
```
Comment (hjvedvik) [inline .gitignore:6]: Can you add this in a separate PR instead? I'm sorry if being too picky here, but I just want to have the PR's a bit organized :)

Comment (sergiocarracedo) [reply .gitignore:6]: Yes, no worries. It's a good idea ;)

**Pull request merged**